### PR TITLE
transform-b​​ox: added from w3c specification

### PR DESCRIPTION
**Pull request by freemedom**
### Description

The case for non-SVG elements(elements with associated CSS layout box) was not explained.
https://w3c.github.io/csswg-drafts/css-transforms/#valdef-transform-box-view-box

### CODE REVIEW

**Comment by freemedom** [comment]
> Preview URLs
> 
> * [/en-US/docs/Web/CSS/transform-box](https://pr25137.content.dev.mdn.mozit.cloud/en-US/docs/Web/CSS/transform-box)

Preview URL will not refresh after adding new commits. I'm not sure if the format is right.

File: files/en-us/web/css/transform-box/index.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ The `transform-box` property is specified as one of the keyword values listed be
   - : The stroke bounding box is used as the reference box.
 - `view-box`
   - : The nearest {{Glossary("SVG")}} viewport is used as the reference box. If a {{SVGAttr("viewBox")}} attribute is specified for the SVG viewport creating element, the reference box is positioned at the origin of the coordinate system established by the `viewBox` attribute, and the dimension of the reference box is set to the width and height values of the `viewBox` attribute.
+  - : For elements with associated CSS layout box, the used value for `fill-box` is `content-box` and for `stroke-box` and `view-box` is `border-box`.
```
Comment (Josh-Cena) [inline files/en-us/web/css/transform-box/index.md:44]: 1. This should not be part of the list because it's not a description for the `view-box` item (and the current syntax is invalid anyway; a definition item can only have one description starting with `- :`)
2. I find this sentence extremely hard to parse. Is it talking about default values?

**Comment by teoli2003** [comment]
> > Preview URLs
> > 
> > * [/en-US/docs/Web/CSS/transform-box](https://pr25137.content.dev.mdn.mozit.cloud/en-US/docs/Web/CSS/transform-box)
> 
> Preview URL will not refresh after adding new commits. I'm not sure if the format is right.

Preview URLs do refresh a few minutes after adding new commits (if the tests passed).

File: files/en-us/web/css/transform-box/index.md (hunk 1 above)
Comment (freemedom) [reply files/en-us/web/css/transform-box/index.md:44]: I'm also not quite sure what "associated CSS layout box" means, but I think it basically means that the Initial value of a transform-box is a view-box, which uses a border-box for non-SVG elements.
(Because when I looked at the default value of transform-box, there was only a description of **svg elements** underneath the view-box. After looking at the w3c, I found an additional description that wasn't written in the mdn. So I think it is necessary to add this content to mdn.)
I don't quite understand the mdn format, you can modify it, thank you.

Comment (estelle) [reply files/en-us/web/css/transform-box/index.md]: @freemedom I made suggested changes to the PR, but since I edited lines you did not edit, I had to commit it directly to the PR. Are my edits what you meant?

Comment (freemedom) [reply files/en-us/web/css/transform-box/index.md]: @estelle Yes, that's what I meant. Thank you.

**Review by estelle — APPROVED** [review]
Thank you. Also, congratulations on your first merged MDN content PR. Welcome to he team!
